scripts/install_environment.sh

```diff
@@ -87,6 +89,19 @@ function is_libvirtd_tcp_socket_enabled_and_running() {
     fi
 }
 
+function allow_libvirt_cross_network_traffic() {
+    # Flush LIBVIRT_FWI chain each time the network configuration is being updated
```

Comment:
```suggestion
    # Flush LIBVIRT_FWI chain each time the network configuration is being updated
    # We do this because .... needs to ... for ....
```